src/components/AutoAdvanceToggle/AutoAdvanceToggle.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { fireEvent, render, screen } from '@testing-library/react';
+import AutoAdvanceToggle from "./AutoAdvanceToggle";
+import manifest from '@TestData/lunchroom-manners';
+import autoAdvanceManifest from '@TestData/multiple-canvas-auto-advance';
+import { withManifestProvider } from '../../services/testing-helpers';
+
+describe('AutoAdvanceToggle', () => {
+  describe('with manifest without auto-advance behavior', () => {
+    beforeEach(() => {
+      const NonAutoManifest = withManifestProvider(AutoAdvanceToggle, {
+        initialState: { manifest, canvasIndex: 0 },
```

Comment:
When I ran the test suite locally I got the following in the console;
```
    console.error
      Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components
          at input
          at label
          at div
          at AutoAdvanceToggle
```
This is triggered by not initializing `autoAdvance` in the `initialState` object passed into the state when setting up the component for testing. Setting `autoAdvance: false` in `initialState` gets rid of the warning.
This is not causing any tests to fail and I wanted to make a note of it.